packages/vite-plugin-cloudflare/src/miniflare-options.ts

```diff
@@ -197,26 +195,15 @@ export function getDevMiniflareOptions(
 	viteDevServer: vite.ViteDevServer
 ): MiniflareOptions {
 	const resolvedViteConfig = viteDevServer.config;
+	const logger = new ViteMiniflareLogger(resolvedViteConfig);
```

Comment:
There may be good reason for moving this here but if it's possible to use `resolvedViteConfig.logger` directly then that would be preferable. Logging about the metafiles isn't really related to Miniflare.

Reply:
It's a different kind of logger with different methods, so it needs to be wrapped. The `ViteMiniflareLogger` is a wrapper around the Vite logger that Miniflare understands.

Reply:
Oh! I just "actually" read your comment and I realise that you are right.

"Logging about the metafiles isn't really related to Miniflare." is the key sentence. We just need to wrap the vite logger to pass to the metafile loader, rather than a double wrap.

Reply:
5430df921ca18ed5497f4b19ec58307f08d89ace